### DISCUSSION TITLE
'Classic' is not even as good as' Difficultly Adjust '

### DISCUSSION THREAD

**Discussion by nana123126** [ppy/osu#25903] [Ideas] [answered]
I think the score multiplier of the new version of Lazer's' Classic 'mod may be too low, and in reality, this mod is reverting to a stable version of the gameplay. Such a low score multiplier is not conducive to players trying it out, and even has a lower cost-effectiveness than other mods. At least let players consider it because I think' Diversity Adjust 'is more useful. At least let me choose one from two options, and I will not hesitate to choose it.

**So the score multiplier for the 'Classic' mod should be between 0.7 and 1.0, which is more appropriate.**

A too low value may force some players to play the stable version, as it is on par with the score multiplier of 'Easy' and 'Difficulty Adjust', but without having a greater gaming impact than them (imagine always using OSU with 'No Failure' or 'Difficulty Adjust', which is more different than now?).

Reducing the score multiplier of 'Diversity Adjust' is also not ideal, which means that both the score multiplier of 'Easy' and 'half Time' must be lowered together, making these mods more difficult to use. Therefore, the score multiplier of 'Classic' mods should be increased.

**Comment by KoleckOLP** [comment]
I found a map that is unplayable without classic

https://osu.ppy.sh/beatmapsets/842236#osu/1762423

the HP drain is so bad that even if I don't miss the HP goes down, and any miss or two are a fail,
meanwhile with the classic mod I have no issues finishing the map even if I get like 11 misses.

**Comment by felipemarins** [comment, marked as answer]
It is currently just a placeholder value so it will probably change in the future. The value being considered now for the next release is 0.96.
